apps/docs/versioned_docs/version-3.1.1/architecture.md

```
@@ -0,0 +1,75 @@
+# Architecture
+
+## Overview
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)